content/200-concepts/100-components/01-prisma-schema/06-relations/100-one-to-one-relations.mdx

```diff
@@ -199,7 +199,11 @@ model User {
 
 This restriction was introduced in 2.12.0.
 
-However, you can choose if the side of the relation _with_ a relation scalar should be optional or mandatory. In the following example, `profile` and `profileId` are mandatory. This means that you cannot create a `User` without connecting or creating a `Profile`:
+However, you can choose if the side of the relation _with_ a relation scalar should be optional or mandatory. 
+
+### Mandatory 1-1 relation
+
+In the following example, `profile` and `profileId` are mandatory. This means that you cannot create a `User` without connecting or creating a `Profile`:
```

Comment:
```suggestion
### Non-optional 1-1 relation

In the following example, `profile` and `profileId` are non-optional. This means that you cannot create a `User` without connecting or creating a `Profile`:
```

"Mandatory" seems too aggressive. Suggestions:
- "Required"
- "Non-optional"
Feel free to select which one is accurate and sounds okay

Reply:
I would prefer either `Mandatory` or `Required`. `Non-optional` doesn't sound right.